pkgs/development/python-modules/bpylist2/default.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  poetry-core,
+  pytest,
+  pytest-pylint,
+  pytest-pycodestyle,
+  pytest-mypy
+}:
+
+buildPythonPackage {
+  pname = "bpylist2";
+  version = "4.1.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "parabolala";
+    repo = "bpylist2";
+    rev = "ddb89e0b0301c6b298de6469221d99b5fe127b58";
+    hash = "sha256-OBwDQZL5++LZgpQM96tmplAh1Pjme3KGSNFTKqKUn00=";
+  };
+
+  build-system = [ poetry-core ];
+
+  patches = [ ./pylint-fix.patch ];
+
+  pythonImportsCheck = [ "bpylist2" ];
+  nativeCheckInputs = [
+    pytest-pylint
+    pytest-pycodestyle
```

Comment:
```suggestion
```
We don't want to check code format in Nixpkgs.

Reply:
Not providing them would cause the following build failure:
```
       > Running phase: installCheckPhase
       > ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
       > pytest: error: unrecognized arguments: --pycodestyle --pylint --pylint-rcfile=pylint.rc
       >   inifile: /build/source/setup.cfg
       >   rootdir: /build/source
       > 
```

Reply:
commonly such arguments are patched out of pyproject.toml or setup.cfg or wherever they are defined in postPatch